calculator.rb

```diff
@@ -0,0 +1,71 @@
+def add(num1, num2)
+  num1 + num2
+end
+
+def subtract(num1, num2)
+  num1 - num2
+end
+
+def divide(num1, num2)
+  if num2 != 0
+    num1 / num2
+  else
+    puts "What are you doing? You can't divide by zero!"
+    return nil
+  end
+end
+
+def multiply(num1, num2)
+  num1 * num2
+end
+
+def verify(input)
+  input = gets.chomp
+  case input 
+  when "0"
```

Comment:
I like that you've split this functionality out as a separate method - good organization.

However, it seems a little odd that you take `input` as a parameter, and then immediately re-assign it with `gets.chomp`. Do you really need a parameter here?